### PR TITLE
[zh-cn]: sync translation for Picture-in-Picture API & add translation for `ShadowRoot.pictureInPictureElement`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

part of the https://github.com/mdn/translated-content/issues/23557

~depends on https://github.com/mdn/content/pull/35887~

### CODE REVIEW

File: files/zh-cn/web/api/picture-in-picture_api/index.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
 ---
-title: 画中画（Picture-in-Picture）API
+title: 画中画 API
 slug: Web/API/Picture-in-Picture_API
 l10n:
-  sourceCommit: 7c1f91ddd9cd584b041d18a916ab850a226824ad
+  sourceCommit: aa8fa82a902746b0bd97839180fc2b5397088140
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/picture-in-picture_api/index.md:5]:
```suggestion
  sourceCommit: bb4d21e3c6e71db4e0ba983a450d6ed628e82670
```

Comment (skyclouds2001) [inline files/zh-cn/web/api/picture-in-picture_api/index.md:5]:
```suggestion
  sourceCommit: bb4d21e3c6e71db4e0ba983a450d6ed628e82670
```

File: files/zh-cn/web/api/shadowroot/pictureinpictureelement/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: ShadowRoot：pictureInPictureElement 属性
+slug: Web/API/ShadowRoot/pictureInPictureElement
+l10n:
+  sourceCommit: f2f9346c0c0e9f6676f2df9f1850933e274401de
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/shadowroot/pictureinpictureelement/index.md:5]:
```suggestion
  sourceCommit: bb4d21e3c6e71db4e0ba983a450d6ed628e82670
```

File: files/zh-cn/web/api/picture-in-picture_api/index.md [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
 slug: Web/API/Picture-in-Picture_API
 l10n:
-  sourceCommit: 7c1f91ddd9cd584b041d18a916ab850a226824ad
+  sourceCommit: bb4d21e3c6e71db4e0ba983a450d6ed628e82670
 ---
 
 {{DefaultAPISidebar("Picture-in-Picture API")}}
 
-**画中画 API** 允许网站总是在其他窗口之上创建一个浮动的视频，以便用户在其他内容站点或者设备上的应用程序交互时可以继续播放媒体。
+**画中画 API（Picture-in-Picture API）**允许网站总是在其他窗口之上创建一个浮动的视频，以便用户在其他内容站点或者设备上的应用程序交互时可以继续播放媒体。
```
Comment (yin1999) [inline files/zh-cn/web/api/picture-in-picture_api/index.md:10]: image


```suggestion
**画中画 API**（Picture-in-Picture API）允许网站总是在其他窗口之上创建一个浮动的视频，以便用户在其他内容站点或者设备上的应用程序交互时可以继续播放媒体。
```

File: files/zh-cn/web/api/picture-in-picture_api/index.md [hunk 4]
```diff
@@ -1,20 +1,22 @@
...
 
-> **备注：** [文档画中画 API](/zh-CN/docs/Web/API/Document_Picture-in-Picture_API) 扩展了画中画 API，使其能够将*任意* HTML 内容（而不仅仅是视频）呈现在始终置顶的窗口中。
+> **备注：** [文档画中画 API（Document Picture-in-Picture API）](/zh-CN/docs/Web/API/Document_Picture-in-Picture_API) 扩展了画中画 API，使其能够将*任意* HTML 内容（而不仅仅是视频）呈现在始终置顶的窗口中。
```
Comment (yin1999) [inline files/zh-cn/web/api/picture-in-picture_api/index.md:12]:
```suggestion
> **备注：** [文档画中画 API](/zh-CN/docs/Web/API/Document_Picture-in-Picture_API) 扩展了画中画 API，使其能够将*任意* HTML 内容（而不仅仅是视频）呈现在始终置顶的窗口中。
```

File: files/zh-cn/web/http/headers/permissions-policy/picture-in-picture/index.md [hunk 5]
```diff
@@ -1,24 +1,28 @@
...
+l10n:
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
 ---
 
 {{HTTPSidebar}} {{SeeCompatTable}}
 
-HTTP 标头 {{HTTPHeader("Permissions-Policy")}} 的 `picture-in-picture` 指令表示是否允许当前的文档通过相应的 API 以画中画模式播放视频。
+HTTP {{HTTPHeader("Permissions-Policy")}} 标头 `picture-in-picture` 指令控制是否允许当前文档以{{domxref("Picture-in-Picture API", "画中画", "", "nocode")}}模式播放视频。
+
+具体而言，如果定义的策略阻止使用此功能，则 {{domxref("HTMLVideoElement.requestPictureInPicture()")}} 调用将抛出类型为 `SecurityError` 的 {{domxref("DOMException")}} 错误。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/permissions-policy/picture-in-picture/index.md:12]:
```suggestion
具体而言，如果定义的策略阻止使用此特性，则 {{domxref("HTMLVideoElement.requestPictureInPicture()")}} 调用将抛出类型为 `SecurityError` 的 {{domxref("DOMException")}} 错误。
```

File: files/zh-cn/web/http/headers/permissions-policy/picture-in-picture/index.md [hunk 6]
```diff
@@ -1,24 +1,28 @@
...
 
 ```http
-Permissions-Policy: picture-in-picture <allowlist>;
+Permissions-Policy: picture-in-picture=<allowlist>;
 ```
 
-- \<allowlist>
-  - : 允许使用该功能的来源列表。参见 [`Permissions-Policy`](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#语法)。
+- `<allowlist>`
+  - : 允许使用该功能的来源列表。有关更多详细信息，参阅 [`Permissions-Policy` > 语法](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#语法)。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/permissions-policy/picture-in-picture/index.md:21]:
```suggestion
  - : 允许使用该特性的来源列表。有关更多详细信息，参阅 [`Permissions-Policy` > 语法](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#语法)。
```

File: files/zh-cn/web/http/headers/permissions-policy/picture-in-picture/index.md [hunk 7]
```diff
@@ -31,5 +35,4 @@ Permissions-Policy: picture-in-picture <allowlist>;
 ## 参见
 
 - {{HTTPHeader("Permissions-Policy")}} 标头
-- [Permission Policy](/zh-CN/docs/Web/HTTP/Feature_Policy)
-- [Using Permission Policy](/zh-CN/docs/Web/HTTP/Feature_Policy/Using_Feature_Policy)
+- [Permission Policy](/zh-CN/docs/Web/HTTP/Permissions_Policy)
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/permissions-policy/picture-in-picture/index.md:38]:
```suggestion
- [权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)
```

File: files/zh-cn/web/api/picture-in-picture_api/index.md [hunk 8]
```diff
@@ -70,7 +72,7 @@ _画中画 API 定义了三个事件，它们可以用来检测画中画模式
```
Comment (yin1999) [inline files/zh-cn/web/api/picture-in-picture_api/index.md:72]: 第 65 行，翻译成“添加控件”会不会更好些（包括第 67 行的）

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**